client/my-sites/checkout/concierge-session-nudge/style.scss

```diff
@@ -1,3 +1,85 @@
+.concierge-session-nudge {
+	b {
+		font-weight: 700;
+	}
+}
+.concierge-session-nudge__column-pane {
+	display: flex;
+	flex-flow: row;
+}
+
+.concierge-session-nudge__column-content {
+	padding-right: 10px;
+	@include breakpoint( '<960px' ) {
+		padding-right: 0;
+	}
+}
+
+.concierge-session-nudge__column-doodle {
+	width: 150px;
```

Comment:
I don't believe this `width` setting is working as the column is shrinking. Please stop the column from shrinking with `flex-shrink`. And the correct width size is 120px.